### ISSUE TITLE
[Question] Is it possible to let maxwell connect to multiple mysql hosts?

### ISSUE THREAD

**Issue by jingchima-appf** [zendesk/maxwell#1107]
We have many database slices, i.e. many hosts for mysql. Each slice has its own hostname. Instead of running maxwell too many times, is it possible to let one maxwell instance connect to multiple slices? Thanks!

**Comment by osheroff** [comment]
no, this is not currently possible.  It's been on my mental list of possible features for awhile (multiple maxwells inside one jvm), but I've never gotten around to doing it.  It might be somewhat easy given embedded-maxwell modes, but there's ramifications I haven't thought through (one server having troubles takes down all the maxwells, or they have to restart, dealing with different configurations, etc).....

**Comment by nipunphutela** [comment]
Hi, osheoff

one maxwell service can read binlogs from multiple MYSQL hosts?

**Comment by gaurav711** [comment]
Hi, osheoff

please let us know if it is possible to have multiple replication hosts for a single maxwell instance.
